scheduler/scheduler-server/build.gradle

```diff
@@ -15,6 +15,8 @@ dependencies {
     compile 'joda-time:joda-time:2.9.1'
     compile 'org.apache.commons:commons-lang3:3.4'
 
+    compile 'org.fusesource.jdbm:jdbm:2.0.1'
```

Comment:
How did you choose jdbm version? There are many forks/version out there.

Reply:
I matched the version already used in scheduler/scheduler-smartproxy-common

Reply:
Ok, make sense.